docs/deploying-identity-provider.md

```diff
@@ -0,0 +1,73 @@
+# Deploying an Identity Provider for Fulcio Integration
```

Comment:
Looks good, but I would add some notes how this is for the PGS, as it being in docs, some might expect this to be guidance for private sigstore deployments (I guess it could be). I did not figure out if was the former until I reached the end (and it spoke of deploying to staging).